persistence.endpoint.http/src/external_task/external_task_controller.ts

```diff
@@ -49,7 +49,7 @@ export class ExternalTaskController extends BaseController implements HttpContro
     const identity = request.identity;
     const workerId = request.params.worker_id;
     const externalTaskId = request.params.external_task_id;
-    const lockExpirationTime = request.query.lock_expiration_time;
+    const lockExpirationTime = typeof request.query.lock_expiration_time === 'string' ? new Date(request.query.lock_expiration_time) : undefined;
```

Comment:
Hier geht das einfacher.

```suggestion
    const lockExpirationTime =  request.query.lock_expiration_time as string;
```